tests/tests.rs

```diff
@@ -106,6 +137,50 @@ fn run_iv(group: &str, name: &str, path: &str, input: &[u8], output_ext: &str) {
     .unwrap();
 }
 
+/// Removes any snapshots that don't have a corresponding test associated with
+/// it
+fn remove_orphan_snapshots(path: &'static str, names: &[String]) {
+  let Ok(entries) = fs::read_dir(path) else {
+    println!("Cannot read path: {}", path);
+    return;
+  };
+
+  let mut orphans = false;
+  for entry in entries {
+    let Ok(entry) = entry else {
+      continue;
```

Comment:
if you remove the semicolon here (and on the other `continue`s), rustfmt should collapse these to a single line, which I think looks a bit nicer

Reply:
very neat